.github/workflows/build_recipes.yaml

```diff
@@ -99,7 +99,7 @@ jobs:
         shell: bash -l {0}
         run: |
           micromamba activate ci-env
-          python -m pip install git+https://github.com/emscripten-forge/empack@main              --no-deps --ignore-installed
```

Comment:
How do we proceed to merge the PRs?

It seems `empack` needs `recipes` to contain the config file before we can release it, and this PR needs empack to be released.

Should we split this PR in two and include the config file only first?

Reply:
@martinRenou no empack does not need this. empack just expects **some path to a config** in the cli.

so we could merge this PR with 
```
python -m pip install git+https://github.com/emscripten-forge/empack@config            --no-deps --ignore-installed
```
then merge https://github.com/emscripten-forge/empack/pull/30

then we change from `config` to `main` and make a new PR
```
python -m pip install git+https://github.com/emscripten-forge/empack@config            --no-deps --ignore-installed
```
this should then not break anything

Reply:
Would it be possible to make a non-breaking empack change?

Maybe by having a default config file pointing to the one from this PR? So https://raw.githubusercontent.com/emscripten-forge/recipes/main/empack_config.yml once this is merged

Reply:
the empack changes also change the CLI itself, so instead of `empack pack python core` it is `empack pack env ...` 

so I would prefer the breaking changes...

cant we make a major version increment and constraint xeus-lite to empack <1.x.x  until we also update xeus-lite?

Reply:
Yes we should do it. I'll update xeus-python lite and make a release right away.